### PR TITLE
Display the sample tweets in the front-end

### PR DESCRIPTION
**Pull request by JeremyLi28**
add tweet and about page

### CODE REVIEW

File: neo/app/views/index.scala.html [hunk 1]
```diff
@@ -7,16 +7,31 @@
 
   <time-series id="chart"></time-series>
 
-  <div id='sidebar' ng-init="click = -1" ng-class="{toggled: click === 1}" ng-click="click = click*-1">
+  <div id='sidebar' ng-init="click = -1; tab=3" ng-class="{toggled: click === 1}" ng-click="click = click*-1">
```
Comment (JavierJia) [inline neo/app/views/index.scala.html]: what's the meaning of `tab=3`?

Comment (JavierJia) [inline neo/app/views/index.scala.html]: never mind, got it now.

Comment (JeremyLi28) [inline neo/app/views/index.scala.html]: I think may be this angular way cause the wield problem, let me have a try use the traditional way

File: neo/app/views/index.scala.html [hunk 2]
```diff
@@ -7,16 +7,31 @@
...
+          <p><b>Cloudberry</b> is a research prototype to support interactive analytics and visualization of large amounts of spatial-temporal data. </p>
+          <p> Basic Information: </p>
+          <ul>
+            <li>Data set: Tweets</li>
+            <li>Number of records: 18,953,055</li>
+            <li>Collection period: From 2016-03-31 to 2016-04-07</li>
+            <li>Total data size: 17G bytes</li>
+            <li><a href="https://github.com/ISG-ICS/cloudberry">Source code</a></li>
+          </ul>
+          <p>The backend is running the big data management system <b>Apache AsterixDB</b> to support large compute clusters. For questions and comments, please contact <b>@Html("isg-cloudberry@ics.uci.edu")</b></p>
```
Comment (JavierJia) [inline neo/app/views/index.scala.html]: please add the ref `http://asterixdb.apache.org/` to `Apache AsterixDB`

File: neo/app/views/index.scala.html (hunk 1 above)
Comment (JavierJia) [inline neo/app/views/index.scala.html]: this blog may help: https://blog.twitter.com/2014/rendering-tweets-with-angularjs-and-nodejs

File: neo/app/views/index.scala.html (hunk 2 above)
Comment (JeremyLi28) [inline neo/app/views/index.scala.html]: Done

**Comment by JavierJia** [comment]
LGTM

**Pull request merged**